plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/DeltaLakeConfig.java

```diff
@@ -36,6 +36,9 @@
 import static java.util.concurrent.TimeUnit.DAYS;
 import static java.util.concurrent.TimeUnit.SECONDS;
 
+
+@DefunctConfig({
+        "delta.experimental.ignore-checkpoint-write-failures"})
```

Comment:
nit: fmt this in a way so that adding new entry doesn't require changing any existing  lines

```suggestion
        "delta.experimental.ignore-checkpoint-write-failures",
})
```

Reply:
so I can't do that because checkstyle complains with compact_no_array and without array i can't add comma